client/src/model/backend/gitlab/cleanLog.ts

```diff
@@ -0,0 +1,31 @@
+const cleanLog = (log: string): string => {
+  if (!log) return '';  
+  
+  let logCache = log.replace(
+    // eslint-disable-next-line no-control-regex
+    /\u001b\[[0-9;]*[mK]/g, 
+    ''
+  );
+  
+  logCache = logCache.replace(
+    // eslint-disable-next-line no-control-regex
+    /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g,
+    ''
+  );
+  
+  const lines = logCache.split('\n');
+  const cleanedLines = lines.map(line => {
+    if (line.match(/^section_(start|end):[0-9]+:[a-zA-Z0-9_-]+$/)) {
+      return '';
+    }
+    return line
+      .replace(/section_start:[0-9]+:[a-zA-Z0-9_-]+/g, '')
```

Comment:
@Microchesst 
Can you give names to the regex strings in this file? For example,

```typescript
SECTION_START = 'section_start:[0-9]+:[a-zA-Z0-9_-]+'
```

You can place all these constants between lines - 1 and 2 of this file.